packages/onchainkit/src/identity/components/SocialPlatformDetails.tsx

```diff
@@ -8,37 +8,49 @@ export type SocialPlatform = 'twitter' | 'github' | 'farcaster' | 'website';
 
 export const PLATFORM_CONFIG: Record<
   SocialPlatform,
-  { href: (value: string) => string; icon: React.ReactNode }
+  {
+    href: (value: string) => string;
+    icon: React.ReactNode;
+    label: (value: string) => string;
```

Comment:
nit: label is a bit ambiguous, so if these are always aria-labels, would be nice to use "aria-label" in the type for clarity